wallet/wallet.sol

```diff
@@ -52,6 +52,7 @@ contract multiowned {
     // constructor is given number of sigs required to do protected "onlymanyowners" transactions
     // as well as the selection of addresses capable of confirming them.
     function multiowned(address[] _owners, uint _required) {
+        if (_required > 1 && _owners.length < _required -1) throw;
```

Comment:
@sunny-g I believe the -1 is because _owners does not include the msg.sender which automatically becomes an owner as well. so if _owners.length is 2 then _required can be 3 (2 owners provided + the msg.sender creating the contract). 

The code should be updated to make this clearer, through comments or by doing a +1 to the owners instead.